distribution/docs/src/main/resources/content/_integrating/_endpoints/catalog-rest-endpoint.adoc

```diff
@@ -185,108 +129,163 @@ The following is a successful post of well-formed XML data sent to the Catalog R
+
+. Make a read request to the REST URL specifying the catalog id.
+. Add a transform query parameter to the end of the URL specifying the shortname of the transformer to be used (e.g., `transform=kml`).
+
+[NOTE]
+====
+Not all installations will have all transformers installed. Contact ${branding} administrator for available transformers.
+====
 
-.Missing XML Tag Example
+.Read Request Example
+[source,javascript,linenums]
 ----
-<metacard xmlns="urn:catalog:metacard" xmlns:gml="http://www.opengis.net/gml" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:smil="http://www.w3.org/2001/SMIL20/" xmlns:smillang="http://www.w3.org/2001/SMIL20/Language" gml:id="3a59483ba44e403a9f0044580343007e">
-  <type>ddf.metacard</type>
-  <string name="title">
-    <value>Test REST Metacard</value>
-  </string>
-  <string name="description">
-    <value>Vestibulum quis ipsum sit amet metus imperdiet vehicula. Nulla scelerisque cursus mi.</value>
-  </string>
-</metacard>
+${public_url}/services/catalog/<metacardId>
 ----
 
-Metacard is created successfully
+//https://github.com/codice/ddf/issues/4374
+//.Read Success Response Example
+//[source,javascript,linenums]
+//----
+//tbd
+//----
 
-.Request with Non-XML Data
-The following is an example request with non-XML data sent to the Catalog ReST endpoint.
-
-.Non-XML data Example
+.Metacard Transform Request Example
+[source,http]
 ----
-title: Non-XML title
-id: abc123
+${public_url}/services/catalog/<metacardId>?transform=<TRANSFORMER_ID>
 ----
 
-Metacard will be created and the content will stored in the `metadata` field.
+[TIP]
+====
+Transforms also work on read operations for metacards in federated sources.
+\${secure_url}/services/catalog/sources/<sourceId>/<metacardId>?transform=<TRANSFORMER_ID>
+====
 
-.Request with Invalid Transform
-Testing valid data with an invalid `transform=invalid` appended to the POST URL: `{public_url}/services/catalog?transform=blah`
+See <<{architecture-prefix}available_metacard_transformers,Metacard Transformers>> for details on metacard transformers.
+
+.Read Error Response Examples
 
-.Valid data with an invalid `?transform=invalid`
+.Request with Invalid Transform
+If a request specifies a transformer that is invalid, unsupported, or not configured,
+${branding} will return an HTTP 400 and the following response body.
+[source,html]
 ----
-<?xml version="1.0" encoding="UTF-8"?>
-<metacard xmlns="urn:catalog:metacard" xmlns:gml="http://www.opengis.net/gml" xmlns:xlink="http://www.w3.org/1999/xlink" xmlns:smil="http://www.w3.org/2001/SMIL20/" xmlns:smillang="http://www.w3.org/2001/SMIL20/Language" gml:id="3a59483ba44e403a9f0044580343007e">
-  <type>ddf.metacard</type>
-  <string name="title">
-    <value>Test REST Metacard</value>
-  </string>
-  <string name="description">
-    <value>Vestibulum quis ipsum sit amet metus imperdiet vehicula. Nulla scelerisque cursus mi.</value>
-  </string>
-</metacard>
+<pre>Error while storing entry in catalog: </pre>
 ----
+'''
```

Comment:
❓ According to [this asciidoc cheat sheet](https://github.com/powerman/asciidoc-cheatsheet#text) should `'''` be `''''` instead?

Reply:
I guess it's a minimum of three, but can be any arbitrary number beyond that: 

https://asciidoctor.org/docs/user-manual/#comparison-by-example